splot/_viz_value_by_alpha.py

```diff
@@ -0,0 +1,36 @@
+import geopandas as gpd
+from shapely.geometry import Polygon
+import matplotlib
+
+"""
+Creating Value by Alpha maps
+
+"""
+
+__author__ = ("Stefanie Lumnitz <stefanie.lumitz@gmail.com>")
+
+
+def value_by_alpha(x, alphay, gdf):
+    
+
+
+
+def value_by_alpha(x='attribute', y='alpha_attribute', c, data=geodataframe):
```

Comment:
This method name is picked in L.13, no?

Reply:
sorry, I should have added `value_by_alpha` functionality is dealt with separately in the next pr #28